packages/react/src/FormControl/FormControl.figma.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import figma from '@figma/code-connect'
+import FormControl from '../FormControl'
+
+figma.connect(
+  FormControl.Label,
+  'https://www.figma.com/design/GCvY3Qv8czRgZgvl1dG6lp/Primer-Web?node-id=34462-69137&t=WXa19NDaeGGhlbLq-4',
+  {
+    props: {
+      label: figma.textContent('Label'),
+    },
+    example: ({label}) => <FormControl.Label>{label}</FormControl.Label>,
```

Comment:
Do you have a preference between disabling the lint rule for these examples specifically or wrapping them in a FormControl? 👀

Reply:
Yes, we need to disable them, as they are nested in other components, that wrap everything in a Form.Control already.